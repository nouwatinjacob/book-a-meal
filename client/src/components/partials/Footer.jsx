import React from 'react';

const Footer = () => (
  <footer>
    <div className='copyright'>
      <p>&copy; 2018 Book-A-Meal <i>designed by</i> Jacob Nouwatin</p>
    </div>
  </footer>
);

export default Footer;
