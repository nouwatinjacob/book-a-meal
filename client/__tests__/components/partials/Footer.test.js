import React from 'react';
import {
  shallow
} from 'enzyme';
import Footer from '../../../src/components/partials/Footer.jsx';

describe('Footer component', () => {
  const setup = () => shallow(<Footer />);

  it('should render Footer component correctly', (done) => {
    const wrapper = setup();
    expect(wrapper).toMatchSnapshot();

    done();
  });
});