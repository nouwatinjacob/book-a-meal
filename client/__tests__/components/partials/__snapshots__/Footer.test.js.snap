// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render Footer component correctly 1`] = `
<footer>
  <div
    className="copyright"
  >
    <p>
      Â© 2018 Book-A-Meal 
      <i>
        designed by
      </i>
       Jacob Nouwatin
    </p>
  </div>
</footer>
`;
